draft-ietf-rats-architecture.md

```diff
@@ -1188,64 +1188,80 @@ in terms of time(RX)-time(RG), then the Relying Party can check
 
 ## Example 3: Timestamp-based Background-Check Model Example
 
+The following example illustrates a hypothetical Background-Check Model solution that uses timestamps and requires roughly synchronized clocks between the Attester, Verifier, and Relying Party.
+
+~~~~
+   .----------.         .---------------.              .----------.
+   | Attester |         | Relying Party |              | Verifier |
+   '----------'         '---------------'              '----------'
+     time(VG)                   |                           |
+           |                    |                           |
+           ~                    ~                           ~
+           |                    |                           |
+     time(EG)                   |                           |
+           |----Evidence------->|                           |
+           |    {time(EG)}   time(ER)--Evidence{time(EG)}-->|
+           |                    |                        time(RG)
+           |                 time(RA)<-Attestation Result---|
+           |                    |        {time(RX)}         |
+           ~                    ~                           ~
+           |                    |                           |
+           |                 time(OP)                       |
+~~~~
+
+The time considerations in this example are equivalent to those discussed under Example 1 above.
+
+## Example 4: Handle-based Background-Check Model Example
+
 The following example illustrates a hypothetical Background-Check Model
-solution that uses centrally generated identifiers for explicit time-keeping (referred to as "handle" in this example).
-Handles can be qualifying data, such as nonces or signed timestamps. In this example, centrally generated signed timestamps and -- and synchronized clocks between all entities -- are distributed
-in periodic intervals as handles.  If the Attester lacks a source of time based on an absolute timescale, a relative source of time, such as a tick counter can be used, alternatively.  In this example, evidence generation is not triggered at value generation, but at events at which the Attesting Environment becomes of changes to the Target Environment.
+solution that uses centrally generated handles for explicit time-keeping (using local clocks) or implicit time-keeping (using trigger events such as receiving a handle or becoming aware of a changed Claim value).
+Handles are be qualifying data, such as nonces or signed timestamps.
```

Comment:
Qualifying for what?  And we already have other sections for nonces and timestamps, I don't follow what's different about this one (especially why it would combine nonce & timestamp into one example when they're separated in the other examples)

Reply:
Things that are typically qualified are: origin of handle or intend of handle. E.g. if your handle is the hash of an handle that is an external timestamp + local clock time + proof of synchronization of the two => you get TUDA. If your handle is the hash of a received nonce received during subscripton, a relative tick-counter originating from your Attesting Environment, and a reference to a global time-scale => you get Attestation Event Stream Subscription.

In general, qualifying data is used in the creation of evidence. It is a annotation of intend and context to a value. The new Restful Resources is showing a good subset in which very explicit rules enrich nonces so they become qualified data. The construction rules defined there provide a whole new spectrum of meanings/semantics to nonces (which TUDA also does, as does network device description, please see below). It is important to highlight that this "semantic enrichment" is allowed, actually used quite often in the field, and goes beyond what is typically perceived as the use of nonces (we also struggle with that until a few weeks ago, when there still was a ver lob-sided focus on reply prevention in the freshness section). It is quite useful to dedicate at least one example in an Appendix to this topic, I think.

Personally, i thought "qualifying data" is a well known term, such as qualified name or such. So my ask is: is this a general misconception of mine? Think of it as nonces with a bigger scope semantics (which we call handles here). If qualifying data is really not self-explanatory, what would be a better phrasing?